vendor/github.com/bazelbuild/buildtools/CONTRIBUTORS

```diff
@@ -0,0 +1,15 @@
+# People who have agreed to one of the CLAs and can contribute patches.
```

Comment:
Is the prune setting supposed to ignore files like this?

Reply:
no, these files are always be reserved, 
https://github.com/golang/dep/blob/5a7960a4cb82dda7253e8b3f921e394cd3c817d3/gps/prune.go#L157